bedrock/firefox/firefox_details.py

```diff
@@ -23,6 +21,11 @@ class FirefoxDetails(ProductDetails):
             'id': 'osx',
             'file_ext': 'mac.dmg',
         },
+        'Windows 64': {
+            'title': 'Windows 64-bit',
+            'id': 'win64',
+            'file_ext': 'win64.installer.exe',
```

Comment:
NIT: The `file_ext` keys are no longer in use now that `_get_aurora_download_url`is gone. They could be removed I'm pretty sure.